handlers/build.gradle

```diff
@@ -0,0 +1,62 @@
+apply plugin: 'info.solidsoft.pitest'
+
+dependencies {
+    implementation project(':domain')
+    implementation project(':handlers-api')
+    implementation project(':producers-api')
+    implementation project(':utils')
+    implementation project(':services-api')
+    implementation project(':transformers')
+    implementation project(':validators')
+}
+
+sourceSets {
+    testUnit {
+        compileClasspath += sourceSets.main.output
+        runtimeClasspath += sourceSets.main.output
+        java {
+            srcDir file('src/unit-test/java')
+        }
+        resources.srcDir file('src/unit-test/resources')
+    }
+}
+
+configurations {
+    testOutput
+    testOutput.extendsFrom(testUnitImplementation)
+}
+
+task testJar(type: Jar, dependsOn: testClasses) {
+    classifier = 'test'
+    from sourceSets.testUnit.output
+}
+
+artifacts {
+    archives testJar
+}
+
+publishing {
+    publications {
+        "$project.name"(MavenPublication) {
+            groupId project.group
+            artifactId project.name
+            artifact testJar
+            version project.version
+            from components.java
+        }
+    }
+}
+
+tasks.register('unit', Test) {
+    description = "Runs unit tests"
+    group = "Verification"
+    testClassesDirs = sourceSets.testUnit.output.classesDirs
+    classpath = sourceSets.testUnit.runtimeClasspath
+    failFast = true
+}
+
+check.dependsOn unit
+
+test {
+    useJUnitPlatform()
+}
```

Comment:
Is this needed?  It's not present in the transformers or validators build.gradle files.  If it is needed then add it to the transformers and validators build.gradle files.  If it is not needed then remove it from here.

Reply:
removed it from handlers